k9mail/src/main/java/com/fsck/k9/notification/NewMailNotifications.java

```diff
@@ -0,0 +1,166 @@
+package com.fsck.k9.notification;
+
+
+import android.app.Notification;
+import android.support.v4.app.NotificationManagerCompat;
+import android.util.SparseArray;
+
+import com.fsck.k9.Account;
+import com.fsck.k9.K9;
+import com.fsck.k9.K9.NotificationHideSubject;
+import com.fsck.k9.activity.MessageReference;
+import com.fsck.k9.mailstore.LocalMessage;
+
+
+class NewMailNotifications {
```

Comment:
took me a while to understand stacked and summary notifications, how they differ and how they are managed by this class. some documentation would be helpful here